doc/matter/conf.py

```diff
@@ -19,6 +19,7 @@
 
 extensions = [
     "recommonmark",
+    "sphinx.ext.intersphinx",
```

Comment:
```suggestion
```

This is not required, `matter` doesn't reference to any other docset.  `nrf` does, though (see https://github.com/nrfconnect/sdk-nrf/blob/master/doc/nrf/conf.py#L81)